packages/preview/flyingcircus/3.0.0/src/Impl.typ

```diff
@@ -0,0 +1,687 @@
+  if (CoverImg != none) {
+    page(paper: "a4", header: none, footer: none, margin: 0pt)[
+      #image.decode(CoverImg, height: 100%, fit: "stretch")
+    ]
+  }
+  //Create Title Page (and eventual TOC)
+  let onepage(body) = page(
+    //Header is evenly LtR, no text
+    header: context{
+      align(center, cetz.canvas(length: 100%, {
+        import cetz.draw:*
+        set-style(stroke: (paint: black, thickness: 0.75mm))
+        line((-.45, 0), (.45, 0))
+        circle((-.45, 0), radius: 1.5mm, fill: black)
+        circle((.45, 0), radius: 1.5mm, fill: black)
+        for x in range(-440, 430, step: 13){
+          line((x / 1000, -2mm), ((x + 20) / 1000, 2mm))
+        }
+      }))
+    },
+    //Footer is smaller and even LtR
+    footer: context{
+      set align(center)
+      cetz.canvas(length: 100%, {
+        import cetz.draw:*
+        set-style(stroke: (paint: black, thickness: 0.75mm))
+        line((-.25, 0), (.25, 0))
+        circle((-.25, 0), radius: 1.5mm, fill: black)
+        circle((.25, 0), radius: 1.5mm, fill: black)
+        for x in range(-240, 230, step: 13){
+          line((x / 1000, -2mm), ((x + 20) / 1000, 2mm))
+        }
+      })
+    },
+    body,
+  )
+  onepage[
+    #set align(center)
+    //The Big FC
+    #KochFont(size: 75pt)[
+      Flying\
+      #box(rotate(90deg)[#scale(x: -100%)[$ theta.alt $]]) Circus #box(rotate(-90deg)[$ theta.alt $])
+    ]
+     
+    //Title and description
+    #text(size: 24pt)[
+      #Desciption \ 
+      #Title
+    ]
+     
+    //Fancy swirls
+    #text(
+      size: 75pt,
+    )[
+      #box(rotate(-90deg, reflow: true)[#scale(y: 300%, reflow: true)[$ sigma.alt $]]) #box(rotate(-90deg, reflow: true)[#scale(y: -300%, reflow: true)[$ sigma.alt $]]) 
+    ]
+     
+    #v(1fr)
+    //Dedication is optional
+    #Dedication
+    #v(2fr)
+    //And the Author, of course
+    #text(size: 24pt)[
+      #Author
+      #v(1em)
+    ]
+  ]
+  //Style outline
+  // show outline.title: it => {
+  //   text
+  // }
+  show outline.entry.where(level: 1): it => {
+    v(10pt, weak: true)
+    text(size: 20pt, it)
+  }
+   
+  onepage[#align(
+      center,
+      box(
+        width: 80%,
+        outline(depth: 10, indent: 1em, title: KochFont(size: 50pt, fill: white, stroke: black + 1pt)[Contents #h(1fr)]),
+      ),
+    )]
+   
+  //Set up headings (gotta be styled after #outline)
+  show heading: it => {
+    KochFont[#align(left)[#it.body]]
+  }
+  show heading.where(level: 1): it => {
+    KochFont(size: 24pt)[#align(left)[#it.body]]
+    v(-1.3em)
+    line(length: 100%, stroke: 1pt + luma(100))
+  }
+  show heading.where(level: 2): it => {
+    KochFont(size: 24pt)[#align(left)[#it.body]]
+  }
+  show heading.where(level: 3): it => {
+    KochFont(size: 18pt)[#align(left)[#it.body]]
+  }
+   
+  //Reset Page counter to 1, and let's go!
+  counter(page).update(1)
+   
+  body
+}
+
+#let MaybeImage(img, ..args) = if (img != none) {
+  image.decode(img, ..args)
```

Comment:
For the new way of passing images to work correctly you need to update this function to something like:
```suggestion
set image(..args)
img
```

Reply:
Apologies, I made a mistake in manually copying changes. Fixed now.